pymc_marketing/mmm/models/components/saturation.py

```diff
@@ -0,0 +1,82 @@
+from pymc_marketing.mmm.models.components.base import Transformation
+from pymc_marketing.mmm.transformers import (
+    hill_saturation,
+    logistic_saturation,
+    michaelis_menten,
+    tanh_saturation,
+    tanh_saturation_baselined,
+)
+
+
+class SaturationTransformation(Transformation):
```

Comment:
One thing to note here @juanitorduz is that this will take x / spend / impressions to contributions rather directly in order to consolidate with `michaelis_menten` transformation which already has the `alpha` multiplied out. That is by the subclasses include the `beta`

It might be more accurate to name `ContributionTransformation`. 
Have any thoughts here?

Reply:
I'm not sure, I think it's more confusing to think about `contribution` functions. A simple docstring can solve these doubts if they appear!

Users might think... "Oh, these models don't use saturation functions"

Reply:
Think that is fair enough reason!

Reply:
I also think `SaturationTransformation` is more transparent.